helpers/deploymentConfig.ts

```diff
@@ -2623,6 +2630,52 @@ export const globalConfig: NetworkConfig = {
         ],
         rewards: [],
       },
+      {
+        id: "Liquid Staked ETH",
+        name: "Liquid Staked ETH",
+        closeFactor: convertToUnit("0.5", 18),
+        liquidationIncentive: convertToUnit("1.02", 18),
+        minLiquidatableCollateral: convertToUnit("100", 18),
+        vtokens: [
+          {
+            name: "Venus wstETH (Liquid Staked ETH)",
+            asset: "wstETH",
+            symbol: "vwstETH_LiquidStakedETH",
+            rateModel: InterestRateModels.JumpRate.toString(),
+            baseRatePerYear: "0",
+            multiplierPerYear: convertToUnit("0.09", 18),
+            jumpMultiplierPerYear: convertToUnit("0.75", 18),
+            kink_: convertToUnit("0.45", 18),
+            collateralFactor: convertToUnit("0.9", 18),
+            liquidationThreshold: convertToUnit("0.93", 18),
+            reserveFactor: convertToUnit("0.2", 18),
+            initialSupply: convertToUnit(5, 18),
+            supplyCap: convertToUnit(20_000, 18),
+            borrowCap: convertToUnit(2_000, 18),
+            reduceReservesBlockDelta: DEFAULT_REDUCE_RESERVES_BLOCK_DELTA,
+            vTokenReceiver: preconfiguredAddresses.sepolia.VTreasury,
+          },
+          {
+            name: "Venus WETH (Liquid Staked ETH)",
+            asset: "WETH",
+            symbol: "vWETH_LiquidStakedETH",
+            rateModel: InterestRateModels.JumpRate.toString(),
+            baseRatePerYear: "0",
+            multiplierPerYear: convertToUnit("0.035", 18),
+            jumpMultiplierPerYear: convertToUnit("0.8", 18),
+            kink_: convertToUnit("0.8", 18),
+            collateralFactor: convertToUnit("0.9", 18),
+            liquidationThreshold: convertToUnit("0.93", 18),
+            reserveFactor: convertToUnit("0.15", 18),
+            initialSupply: convertToUnit(5, 18),
```

Comment:
We only have 0.5 wstETH tokens in the VTreasury on Sepolia

```suggestion
            initialSupply: convertToUnit(0.5, 18),
```

Reply:
We have enough liquidity now.